tests/Common/CommonSchemaTest.php

```diff
@@ -405,6 +405,27 @@ public function testGetTableNames(array $pdoAttributes): void
         $db->close();
     }
 
+    public function testHasTable(array $pdoAttributes): void
+    {
+        $db = $this->getConnection(true);
+
+        foreach ($pdoAttributes as $name => $value) {
+            if ($name === PDO::ATTR_EMULATE_PREPARES) {
+                continue;
+            }
+
+            $db->getPDO()?->setAttribute($name, $value);
+        }
```

Comment:
Why need to skeep `PDO::ATTR_EMULATE_PREPARES` attribute?
It does not work without this?

Reply:
Not sure exactly why do we need it. The code was taken from the existing test [testGetTableNames()](https://github.com/yiisoft/db/blob/4f9a12a01d93bcffe3e4fecc23c53ff86d70dd43/tests/Common/CommonSchemaTest.php#L381) which is used in tgis new method under the hood.

Reply:
Let's remove this, it should work without this. And need to check that all tests passed successfully